.commitlintrc.ts

```diff
@@ -0,0 +1,83 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+interface PackageJson {
+  name: string;
+  workspaces?: string[];
+}
+
+interface Context {
+  cwd?: string;
+}
+
```

Comment:
I would say let's add a comment explaining why this custom work is necessary, how it basically replaces the `pnpm-scopes` package because we use bun as the package manager.

Reply:
Good shout. Has been added.